test/LibraryManager.Test/Providers/Unpkg/NpmPackageSearchTest.cs

```diff
@@ -13,7 +13,7 @@ public class NpmPackageSearchTest
         [DataTestMethod]
         [DataRow(null, 0)]
         [DataRow("", 0)]
-        [DataRow("poiuytrewq", 0)]
+        [DataRow("plajsfdlkjsfoiuytrewq", 0)]
```

Comment:
Could you use an uppercase letter to make sure that someone can't do that to you in the future (Since uppercase letters aren't allowed in new package names anymore but are valid for existing packages)?

Reply:
After posting that, I'm not realizing this is searching not looking for exact match, so that probably wouldn't matter. Nevermind.